packages/braintree-integration/src/braintree-integration-service.ts

```diff
@@ -92,6 +94,17 @@ export default class BraintreeIntegrationService {
         return this.paypalCheckout;
     }
 
+    async getUsBankAccount() {
+        if (!this.usBankAccount) {
+            this.usBankAccount = Promise.all([
+                this.getClient(),
+                this.braintreeScriptLoader.loadUsBankAccount(),
+            ]).then(([client, usBankAccount]) => usBankAccount.create({ client }));
+        }
+
+        return this.usBankAccount;
+    }
```

Comment:
As far as I see, you are using async here. In this case this method can be updated like this:

```suggestion
    async getUsBankAccount() {
        if (!this.usBankAccount) {
            const client = await this.getClient();
            const usBankAccount = await loadUsBankAccount();
            
            this.usBankAccount = usBankAccount.create({ client });
        }

        return this.usBankAccount;
    }
```